package/build_manager.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+STAGE_DIR=stage
+
+export CHROME_BIN=/usr/bin/google-chrome
+
+rm -rf admin/target
+pushd admin/webapp
+if [[ $# > 0 ]]; then
+    case $1 in
+        -d)
+        mkdir -p /root/.ivy2
+        ln -s /prebuild/manager/cache /root/.ivy2/cache
+        # ln -s /prebuild/manager/node_modules node_modules
+        ;;
+        *)
+        ;;
+    esac
+fi
+npm install --legacy-peer-deps 2>&1
+if [ $? -eq 0 ]; then
+    echo npm package installation SUCCEED
+else
+    npm cache clean --force 2>&1
+    sleep 10
+    npm install 2>&1
+    if [ $? -eq 0 ]; then
+        echo npm package installation SUCCEED
+    else
+        echo ================================
+        echo npm package installation FAILED
+        echo ================================
+        exit 1
+    fi
+fi
+npm run build 2>&1
+if [ $? -eq 0 ]; then
+    echo UI build SUCCEED
+else
+    echo ================================
+    echo UI build FAILED
+    echo ================================
+    exit 1
+fi
+# npm run unittest
+popd
+env JAVA_OPTS="-Xms2g -Xmx3g" sbt admin/assembly
+zip -d admin/target/scala-3.3.4/admin-assembly-1.0.jar rest-management-private-classpath\*
+rm -rf admin/webapp/root/.sass-cache
+
+mkdir -p ${STAGE_DIR}/licenses/ ${STAGE_DIR}/usr/local/bin/ ${STAGE_DIR}/usr/lib/jvm/java-17-openjdk/lib/security/java.security
+cp licenses/* ${STAGE_DIR}/licenses/
+cp cli/cli ${STAGE_DIR}/usr/local/bin/
+cp cli/cli.py ${STAGE_DIR}/usr/local/bin/
+cp -r cli/prog ${STAGE_DIR}/usr/local/bin/
+cp scripts/* ${STAGE_DIR}/usr/local/bin/
+cp java.security ${STAGE_DIR}/usr/lib/jvm/java-17-openjdk/lib/security/java.security
+cp java.security ${STAGE_DIR}/usr/lib64/jvm/java-17-openjdk-17/conf/security/java.security
```

Comment:
I see.  I think it's because the files in /src/stage get overwritten by its next line.  

```
COPY --from=builder /src/stage /
COPY --from=builder /usr/lib64/ /usr/lib64/
```
Any concern if I swap them?

Reply:
I think we can try swapping them to see if it works correctly. We just need to make sure the packages that Java requires under the /usr/lib64/ are included.